tests/runner/loadtest/k6.go

```diff
@@ -152,10 +152,14 @@ type K6 struct {
 	testMemoryRequest string
 	daprMemoryLimit   string
 	daprMemoryRequest string
+	logEnabled        bool
 }
 
 // collectResult read the pod logs and transform into json output.
 func collectResult[T any](k6 *K6, podName string) (*T, error) {
+	if k6.logEnabled {
```

Comment:
Should this have been `!k6.logEnabled` maybe?

Reply:
Not really, the log is disabled by default to not mess up with the test summary output which is sent straight to the stdout.

So when the log is enabled I'm assuming that the summary might not be well-formatted